### DISCUSSION TITLE
How to search only starting with first letters

### DISCUSSION THREAD

**Discussion by ukccatc** [TarekRaafat/autoComplete.js#397] [General]
I need to make search not by all substring included, but first to display words which begin with entered letters. Is that possible?

**Comment by ghost** [comment]
https://tarekraafat.github.io/autoComplete.js/#/how-to-guides?id=results-start-with

**Comment by TarekRaafat** [comment]
This is already covered in the how-to guides: https://tarekraafat.github.io/autoComplete.js/#/how-to-guides?id=results-start-with

Cheers! :)
